### PR TITLE
ECS container instructions

### PR DESCRIPTION
**Pull request by adnanrahic**
Added an integrations page for ECS. We still don't have a logpack for this but I reckon we should jump on it ASAP. It's at `/docs/integration/ecs-logs` located under **Integrations > Logs > ECS**.

I'll go ahead and add UI instructions for this in SC under the Generic Logs App Type. Once I'm done with that, I'll create tasks to create a logpack for this.

localhost_8000_integration_ecs-logs_

### CODE REVIEW

File: docs/integration/ecs-logs.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
+title: Elastic Container Service (ECS) Logs Integration
+description: Whether you're using EC2 or Fargate, forward all container logs to Sematext Cloud. Get insight into your whole ECS Cluster.
+
+By configuring either Logagent for EC2 or AWS Firelens for Fargate, you can forward all your container logs to Sematext Logs and get insight into your whole Elastic Container Service (ECS) cluster in one place!
```
Comment (otisg) [inline docs/integration/ecs-logs.md:4]: to Sematext Logs => to Sematext

File: docs/integration/ecs-logs.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+
+## ECS Logs Quick Start
+
+The ECS Logs Integration collects logs from ECS Tasks and Services running in:
```
Comment (otisg) [inline docs/integration/ecs-logs.md:8]: The ECS Logs Integration ==> The ECS Logs integration

File: docs/integration/ecs-logs.md [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+
+### AWS Firelens
+With Firelens you can route logs to another AWS service, like Firehose, or use Fluentd or Fluent Bit. AWS provides the image for Fluentd / Fluent Bit. You need to configure the output module.
+
+#### 1. Enable Firelens
+In the ECS Task Definition, check a checkbox called Enable FireLens integration. Choose Fluent Bit and AWS will populate the image name for you.
+
+![aws-ecs-firelens-1](../images/integrations/aws-ecs-firelens-1.png)
+
+An additional container called `log_router` will show up.
```
Comment (otisg) [inline docs/integration/ecs-logs.md:32]: Where will it show up?  UI?  Sounds like AWS will launch it?  If so, maybe:
AWS will launch an additional container called `log_router`.

File: docs/integration/ecs-logs.md [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+  "options": {
+    "awslogs-group": "/ecs/ecs-service-name",
+    "awslogs-region": "eu-central-1",
+    "awslogs-stream-prefix": "ecs"
+  }
+}
+```
+
+#### 2. Set up a Lambda function pipeline to collect and forward CloudWatch logs to Sematext
+Once configured, you need to set up a Lambda function to collect these logs from CloudWatch and send them to Sematext. You do this by [following this guide](https://sematext.com/blog/centralized-aws-lambda-logs-kinesis-serverless/). Or, if you already know how to, [here is the code for the Lambda pipeline so you can deploy](https://github.com/sematext/cloudwatch-sematext-aws-lambda-log-shipper) right away.
```
Comment (otisg) [inline docs/integration/ecs-logs.md:74]: Once configure => Once forwarding to CloudWatch is configured

File: mkdocs.yml [hunk 5]
```diff
@@ -135,6 +135,7 @@ pages:
         - Mobile Apps Logs: integration/mobile-apps-logs.md
         - IBM Cloud Kubernetes Logs: integration/ibm-cloud-kubernetes-logs.md
         - RabbitMQ: integration/rabbitmq-logs.md
+        - ECS: integration/ecs-logs.md
```
Comment (otisg) [inline mkdocs.yml:138]: ECS => AWS ECS

**Review by otisg — APPROVED** [review]
Looks good!  Just minor mods needed before merging.

File: docs/integration/ecs-logs.md (hunk 3 above)
Comment (adnanrahic) [reply docs/integration/ecs-logs.md:32]: Yes, in the UI. So this is a better way of saying it:

> AWS will add an additional container called `log_router` to the list of containers in your Task Definition

Comment (otisg) [reply docs/integration/ecs-logs.md:32]: OK

**Review by radu-gheorghe — APPROVED** [review]
LGTM. Awesome work, Adnan!

**Pull request merged**